internal/bminventory/inventory.go

```diff
@@ -2151,6 +2185,22 @@ func (b *bareMetalInventory) processDhcpAllocationResponse(ctx context.Context,
 	return b.clusterApi.SetVipsData(ctx, &cluster, apiVip, ingressVip, dhcpAllocationReponse.APIVipLease, dhcpAllocationReponse.IngressVipLease, b.db)
 }
 
+func (b *bareMetalInventory) processNtpSynchronizerResponse(ctx context.Context, host *models.Host, ntpSynchronizerResponseStr string) error {
+	var (
+		err                     error
+		ntpSynchronizerResponse models.NtpSynchronizationResponse
+	)
+
+	log := logutil.FromContext(ctx, b.log)
+
+	if err = json.Unmarshal([]byte(ntpSynchronizerResponseStr), &ntpSynchronizerResponse); err != nil {
```

Comment:
Super minor, but if you changed this then you could remove the `var` block above as you don't need `err` outside the scope of this conditional.
```suggestion
	if err := json.Unmarshal([]byte(ntpSynchronizerResponseStr), &ntpSynchronizerResponse); err != nil {
```